src/main/java/org/javamodularity/moduleplugin/tasks/StartScriptsMutator.java

```diff
@@ -120,4 +131,16 @@ private static void replaceLibsPlaceHolder(Path path, String libText, String pat
             throw new GradleException("Couldn't replace placeholder in " + path);
         }
     }
+
+    private static void removeClasspathArg(Path path, String original, String replacement) {
```

Comment:
I'd rename this method to sth like `replaceScriptContent` (it's what the method body actually does).

Plus, I'd rename `original` parameter to `regex` (to match [`String.replaceAll`](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#replaceAll-java.lang.String-java.lang.String-)).